fuzzers/tutorial/src/metadata.rs

```diff
@@ -45,9 +45,13 @@ impl<S> Feedback<PacketData, S> for PacketLenFeedback
 where
     S: HasClientPerfMonitor,
 {
+    type FeedbackState = NopFeedbackState;
```

Comment:
Why do we still have the NopFeedbackState if we don't need a feedbackstate anymore?

Reply:
Or is it just not tidied up yet?